nc-news_/src/Components/SingleArticle/ArticleBody.jsx

```diff
@@ -0,0 +1,31 @@
+import apiClient from "../../api";
+import { useEffect, useState } from "react";
+
+const ArticleBody = ({ id }) => {
+  const [article, setArticle] = useState({});
+  useEffect(() => {
+    apiClient
+      .get(`articles/${id}`)
```

Comment:
A bit picky, but if you already have an api.js file, it would be neater to create functions in that file that do the api calls and then invoke the functions in the component